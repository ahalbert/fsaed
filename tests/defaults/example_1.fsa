/foo/ -> /bar/ -> do s/baz/bang/
