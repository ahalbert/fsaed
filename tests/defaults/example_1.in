baz
foo
baz
bar
baz
