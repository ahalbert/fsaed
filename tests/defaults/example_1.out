baz
foo
baz
bar
bang
